client/src/helpers/fhir/fhir.js

```diff
@@ -90,6 +94,37 @@ export const getObservationValue = (obs, defaultValue) => (
   get(obs, 'valueString', get(obs, 'note[0].text', defaultValue))
 )
 
+export const getNoteComment = (serviceRequest) => {
```

Comment:
1. avoid coercion but using triple equality
2. you may create a small function for you empty stuff. Something like (forget about the bad name I'll give):
    `const extractNoteorNull = (rawNote) => !rawNote || rawNote === EMPTY_NOTE_TEXT ? null : rawNote;`